lms/envs/discussionsettings.py

```diff
@@ -1,2 +0,0 @@
-
-DISCUSSION_ALLOWED_UPLOAD_FILE_TYPES = ('.jpg', '.jpeg', '.gif', '.bmp', '.png', '.tiff')
```

Comment:
I couldn't find any usages of `DISCUSSION_ALLOWED_UPLOAD_FILE_TYPES` in the codebase.